levelkdev/swapr-dapp
# Loading swap button

# Summary

Fixes Issue #994 #894

Show a loading state till all the exchanges for swap is fetched.

https://user-images.githubusercontent.com/102727631/170865192-7bd2e40a-ec32-446a-9734-e4bf5f45d442.mp4


  # To Test

Go to swap page => Select both tokens => Enter a value => Loading button should show till all the exchanges are loaded.
